### ISSUE TITLE
Parser Collaboration

### ISSUE THREAD

**Issue by joemcbride** [mkmarek/graphql-dotnetcore#33]
I'm the creator/maintainer of the [graphql-dotnet](https://github.com/graphql-dotnet/graphql-dotnet) project.  I noticed your project a few days ago, its looking pretty great!

One thing that my project has been lacking in a .NET version of the core parser.  Now that you've written one, I have plans to use that in my project.

What I would like to propose though is that we collaborate on it together.  I think it could be beneficial to have the core parser as a standalone Nuget that both of our projects could depend on.  That way when there are updates to the spec or bug fixes we both can benefit.

What do you think?

**Comment by mkmarek** [comment]
Hi,

I totally aree with the benefits. I also think it would be a good idea to separate it into its own nuget package and I would be definetly looking forward to our collaboration on this.

Just now I created a repository where I tried to create a package containing standalone parser and its tests. I reused the approach that Newtonsoft.Json uses to create their multiplatform nuget package.

https://github.com/mkmarek/graphql-dotnet-parser

I also tried a test deploy to myget and it seems that it works nicely.
https://www.myget.org/feed/graphqlcore-ci/package/nuget/GraphQLParser

Also since we both are gonna be dependent on this package then it would make sense to make it more futureproof in a sense that it wouldn't be bound on our user accounts (specifically mine in this case). I would propose moving it into your graphql-dotnet organization.

Then it would be a matter of setting up CI and autodeploy.
